### DISCUSSION TITLE
Deployment in Azure

### DISCUSSION THREAD

**Discussion by ciscocollabguru** [DependencyTrack/dependency-track#1570] [Q&A]
I posed this question in the slack channel but figured I'd put it here too for better visibility.  I'm looking for guidance on deploying dependency track in Azure's container registry.  A simple deployment in my lab using docker on ubuntu worked well but for a larger environment and for robustness, Azure makes more sense.  Has anyone gotten this to work successfully and is there any documentation around that process?

**Comment by jgstechnology** [comment]
We are in the same boat - we have deployed to docker swarm and getting timeouts on the service regularly after about 15-30 seconds. Its causing us a major headache and no idea where to look next.

**Reply by ciscocollabguru** [reply]
I will say for the Azure piece, because there's no ability to map ports, one of my DevOps guys was able to get it spun up in separate container instances that eliminates the need for the mapping of 8081 to 8080.  That said though, doing it via compose _doesn't_ work as intended...in reality, it's something that should be done through AKS anyway...along with a RP in front for TLS.  Still tinkering/testing so i'll post a final answer when I get closer to getting it sorted out for sure.

**Reply by jgstechnology** [reply]
Awesome thanks mate.

**Reply by ciscocollabguru** [reply]
@tfhowells we did finally get it cooking...unfortunately I don't have access to the documentation I had gotten because I'm no longer at the company where we spun it up.  However - the guy on the cloud team that assisted did get back to me and the main issue (we think at least) was persistent storage for the database.  Once we got that part sorted out, it ran fairly well for the little bit we were doing with it.  I want to spin it up at some point in the future just for s&g's so after I do, if there was more to it, I'll post a follow-up here.

HTH!

**Comment by ciscocollabguru** [comment]
My cloud dude eventually did…I forget how he did it though. I’m no longer there so I don’t have the notes on it. If I can get any detail though, I’ll post here for sure.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
